build-system/check-package-manager.js

```diff
@@ -94,23 +85,22 @@ function ensureYarn() {
 }
 
 // Check the node version and print a warning if it is not the latest LTS.
-function performNodeVersionCheck(latestLtsMajorVersion) {
+function performNodeVersionCheck(latestLtsVersion) {
   const nodeVersion = getStdout('node --version').trim();
-  const nodeMajorVersion = nodeVersion.split('.')[0];
-  if (latestLtsMajorVersion === '') {
+  if (latestLtsVersion === '') {
     console.log(yellow('WARNING: Something went wrong. ' +
         'Could not determine latest LTS node version.'));
   }
-  if (latestLtsMajorVersion !== '' &&
-      nodeMajorVersion !== latestLtsMajorVersion) {
-    console.log(yellow('WARNING: Detected node version'),
-        cyan(nodeMajorVersion) +
+  if (latestLtsVersion !== '' && nodeVersion !== latestLtsVersion) {
```

Comment:
Agree with @jridgewell's feedback. However, this might be an opportunity to collapse logic for minimum versions into the `preinstall` script, and use "lts{major}" as the error enforced minimum, and "lts{major.minor.patch}" becomes warnable.

Reply:
@kristoferbaxter Unfortunately, we can't do that because we're seeing errors in our toolchain with versions like `8.9.4`, which satisfy `lts{major}`, but not `lts{major.minor}`.